pkg/api/validation/validation.go

```diff
@@ -30,12 +30,30 @@ import (
 	"github.com/golang/glog"
 )
 
+const minPort int = 1
+const minHostPort int = 0
+const maxPort int = 65536
+
+const qualifiedNameErrorMsg string = "must match regex [" + util.DNS1123SubdomainFmt + " / ] " + util.DNS1123LabelFmt
+const cIdentifierErrorMsg string = "must match regex " + util.CIdentifierFmt
+const isNegativeErrorMsg string = "value must not be negative"
+
+func intervalErrorMsg(lo, hi int) string {
+	return fmt.Sprintf("must be greater than %d and less than %d", lo, hi)
+}
+
+var dnsSubdomainErrorMsg string = fmt.Sprintf("must have at most %d characters and match regex %s", util.DNS1123SubdomainMaxLength, util.DNS1123SubdomainFmt)
+var dnsLabelErrorMsg string = fmt.Sprintf("must have at most %d characters and match regex %s", util.DNS1123LabelMaxLength, util.DNS1123LabelFmt)
+var dns952LabelErrorMsg string = fmt.Sprintf("must have at most %d characters and match regex %s", util.DNS952LabelMaxLength, util.DNS952LabelFmt)
+var pdPartitionErrorMsg string = intervalErrorMsg(0, 255)
+var portRangeErrorMsg string = intervalErrorMsg(minPort, maxPort)
```

Comment:
Given that port numbers are well-known, there's no point in making them variables - I'd just as soon put them here as literals.  Also, the lo arg here is 0, since the error message defines it as non-inclusive :)